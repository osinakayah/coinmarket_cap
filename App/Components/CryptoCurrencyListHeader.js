import React from 'react'
import { View, Text } from 'react-native'
import styles from './Styles/CryptoCurrencyListHeaderStyle'

import {Colors, Fonts} from '../Themes'
import { Col, Grid, Row } from 'react-native-easy-grid'


const CryptoCurrencyListHeader = () => (
  <View style={styles.container}>
    <Grid>
      <Col size={0.5} />
      <Col size={3}>
        <Row>
          <Col size={3}><Text style={{fontFamily: Fonts.type.bold, color: Colors.accentColor, textAlign: 'left'}}>Symbol</Text></Col>
          <Col size={3} />
          <Col size={3}><Text style={[{fontFamily: Fonts.type.bold, textAlign: 'right', color: Colors.accentColor}]}>Price</Text></Col>
        </Row>
        <Row>
          <Col size={6}><Text style={{fontFamily: Fonts.type.bold, color: Colors.accentColor, textAlign: 'left'}}>Name</Text></Col>
          <Col size={3}><Text style={[{fontFamily: Fonts.type.bold, textAlign: 'right', color: Colors.accentColor}]}>% Change</Text></Col>
        </Row>
      </Col>
    </Grid>
  </View>
)

export default CryptoCurrencyListHeader
