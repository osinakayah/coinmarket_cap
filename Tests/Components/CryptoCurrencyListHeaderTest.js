
import 'react-native'
import React from 'react'
import CryptoCurrencyListHeader from '../../App/Components/CryptoCurrencyListHeader'
import renderer from 'react-test-renderer'

test('CenterItem component renders correctly', () => {
  const tree = renderer.create(<CryptoCurrencyListHeader />).toJSON()
  expect(tree).toMatchSnapshot()
})
