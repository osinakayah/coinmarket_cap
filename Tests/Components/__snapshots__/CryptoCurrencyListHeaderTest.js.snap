// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterItem component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#323F52",
      "flex": 1,
      "padding": 7,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      size={0.5}
      style={
        Object {
          "flex": 0.5,
          "flexDirection": "column",
        }
      }
    />
    <View
      size={3}
      style={
        Object {
          "flex": 3,
          "flexDirection": "column",
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
          }
        }
      >
        <View
          size={3}
          style={
            Object {
              "flex": 3,
              "flexDirection": "column",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#FFF",
                "fontFamily": "Nunito-SemiBold",
                "textAlign": "left",
              }
            }
          >
            Symbol
          </Text>
        </View>
        <View
          size={3}
          style={
            Object {
              "flex": 3,
              "flexDirection": "column",
            }
          }
        />
        <View
          size={3}
          style={
            Object {
              "flex": 3,
              "flexDirection": "column",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#FFF",
                  "fontFamily": "Nunito-SemiBold",
                  "textAlign": "right",
                },
              ]
            }
          >
            Price
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
          }
        }
      >
        <View
          size={6}
          style={
            Object {
              "flex": 6,
              "flexDirection": "column",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#FFF",
                "fontFamily": "Nunito-SemiBold",
                "textAlign": "left",
              }
            }
          >
            Name
          </Text>
        </View>
        <View
          size={3}
          style={
            Object {
              "flex": 3,
              "flexDirection": "column",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#FFF",
                  "fontFamily": "Nunito-SemiBold",
                  "textAlign": "right",
                },
              ]
            }
          >
            % Change
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
